test/unit/modules/observer/observer.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('Observer', () => {
     const ob1 = observe(1)
     expect(ob1).toBeUndefined()
     // avoid vue instance
-    const ob2 = observe(new Vue())
+    const ob2 = observe(new Vue('div'))
```

Comment:
div here

Reply:
Do we want to roll this back? Contract is unhappy with no-arg constructor, but maybe that's part of the inconsistency you found in the API. 

Either way, LMK and I'll get it done across the board :)

Reply:
I think we should rollback, it was working before 🤔 
We can make args to be optional

Reply:
Nevermind, saw the other comments. Cleaned!